lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -4289,18 +4286,9 @@ private void createToolBar(UIContainer tofill, SimplePage currentPage) {
 
 		    // add website.
 		    // Are we running a kernel with KNL-273?
-		    Class contentHostingInterface = ContentHostingService.class;
-		    try {
-			Method expandMethod = contentHostingInterface.getMethod("expandZippedResource", new Class[] { String.class });
-			
 			UIOutput.make(tofill, "addwebsite-li");
```

Comment:
Can you outdent these two lines?

Reply:
yeah the indentation is mixed here, lines are using 2 tabs and 4 spaces so decided not to update it to keep history

Reply:
Thanks for refactoring @ern!

I'd like to backport your refactor to our 12.7 instance.  Is there anything I'd need to adjust from your backport?